internal/server/execclient/client.go

```diff
@@ -32,6 +32,10 @@ type Client struct {
 	Stdin         io.Reader
 	Stdout        io.Writer
 	Stderr        io.Writer
+
+	// If set, will cause the server to connect to this specific
+	// instance.
+	InstanceId string
```

Comment:
I'd pull DeploymentId/DeploymentSeq down here too and just like use comments to document they're a `oneof`.

Reply:
👍